### PR TITLE
Replace adjusted text control implementations

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: FueledUtils.xcworkspace/xcshareddata/FueledUtils.xcscmblueprint [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "46B19F3040765B0804498FD4689AE86ED8E3E630" : "FueledUtils\/",
```
Comment (ghost) [inline FueledUtils.xcworkspace/xcshareddata/FueledUtils.xcscmblueprint:14]: this file should be added to `.gitignore` ;)
```
*.xcscmblueprint
```

**Review by ghost — APPROVED** [review]
Looks good to me, the .gitignore stuff can just be done later on

**Review by bastien-fueled — APPROVED** [review]
Looks good @leonty-fueled, thanks for the update!

File: FueledUtils.xcworkspace/xcshareddata/FueledUtils.xcscmblueprint (hunk 1 above)
Comment (ghost) [reply FueledUtils.xcworkspace/xcshareddata/FueledUtils.xcscmblueprint:14]: Oh, I had no idea it was not necessary. But yeah, looks redundant. I thought xcshareddata was only for things like shared schemes and shared breakpoints.

**Pull request merged**